app/main/views/users.py

```diff
@@ -277,6 +278,7 @@ def export_users_for_framework(framework_slug):
             else:
                 application_result = 'pass' if sf.on_framework else 'fail'
             framework_agreement = bool(sf.agreement_returned_at)
+            variations_agreed = ', '.join(sf.agreed_variations.keys()) if sf.agreed_variations else ''
```

Comment:
Actually, looking at it again all the other fields get an empty string if there's no value (e.g. `application_status`, `application_result`, `framework_agreement`) so it seems reasonable to leave blank for no variations agreed either (also bearing in mind that most frameworks don't have any variations, so a column of "None" might look a bit much when nothing would be expected there anyway).